management/update_boptest.py

```diff
@@ -0,0 +1,39 @@
+import distutils.cmd
+import distutils.log
+import os
+import requests
+
+
+class UpdateBoptest(distutils.cmd.Command):
+    """Custom comand for updating the GeoJSON schemas on which this project depends."""
+
+    description = "Download and overwrite latest BOPTEST files from source"
+
+    user_options = [("baseurl=", "u", "base URL from which to download the schemas")]
+
+    def initialize_options(self):
+        self.files_to_download = {
+            'data/categories.json': 'alfalfa_worker/lib/data/categories.json',
```

Comment:
initial list of files that we pull from BOPTEST. need to add others as we move along.